src/components/Todo/Todo.css

```diff
@@ -0,0 +1,49 @@
+.todo-container {
+    width: 20vw;
+    background-color: #0f172a;
```

Comment:
Suggestion: Stay consistent if you are using a variable for some color and a hex for some color.

Reply:
thank you for pointing this out, I somehow missed it

Reply:
thanks, updated